Week1/Section3/Exercise8/Source8.cpp

```diff
@@ -73,16 +73,16 @@ int main()
 	int Jackpot = 0;
 	std::cout << "\nLottery Result: ";
 
-	for (int a : Lottery) 
+	for (int i : Lottery) 
```

Comment:
Remember that these are not indices and should have descriptive names :)

Reply:
Addressed 👌